.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.1.0
+  github-super-linter: circleci/github-super-linter@1.0.1
+
+commands:
+  install_dependencies:
+    steps:
+      - node/install:
+          install-yarn: false
+          node-version: '18.16.0'
+      - run:
+          name: Install node dependencies
+          command: npm install
+  setup_prisma:
+    steps:
+      - run:
+          name: Generate prisma model
+          command: npx prisma generate
+      - run:
+          name: Generate and apply migration
+          command: npx prisma migrate dev
+  setup_docker_compose:
+    steps:
+      - setup_remote_docker:
+          version: 20.10.14
+      - run:
+          name: Install Docker Compose
+          environment:
+            COMPOSE_VERSION: '1.29.2'
+          command: |
+            curl -L "https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o ~/docker-compose
+            chmod +x ~/docker-compose
+            sudo mv ~/docker-compose /usr/local/bin/docker-compose
+      - run:
+          name: Run images of services declared in docker-compose.yml
+          command: docker-compose up -d
+
+jobs:
+  build_and_test:
+    working_directory: ~/repo
+
+    docker:
+      - image: cimg/base:2023.07
+      - image: cimg/postgres:14.1
+
+    steps:
+      - checkout
+      - install_dependencies
+      - setup_prisma
+      - setup_docker_compose
+      - run:
+          name: Run unit tests
+          command: npm run test
+
+  deploy:
+    working_directory: ~/repo
+
+    docker:
+      - image: cimg/base:2023.07
+      - image: cimg/postgres:14.1
```

Comment:
Why do you need postgres for deployment?

Reply:
Thought, I have to deploy the app as well as db for it. With that suggestion I assume db will be hosted on different server?